src/pages/ProhibitorySigns/components/MapLegend/MapLegend.test.tsx

```diff
@@ -9,13 +9,13 @@ describe('MapLegend', () => {
   jest.setTimeout(10000)
 
   it('renders correctly', async () => {
-    const pathToPage = generatePath(getPathTo('HOME'))
+    const pathToPage = generatePath(getPathTo('LICENCE_PLATE_PAGE'))
```

Comment:
Yes, I added `getGeneratedPath` because the code previously used `generatePath(getPathTo('page'))` everywhere. I would like to use `getGeneratedPath` but didn't change that call in all the tests.

Reply:
A new PR is coming that replaces them everywhere.